### PR TITLE
SP-8865 fitting description text for appleTV

### PR DESCRIPTION
**Pull request by Nevazhnovu**

### CODE REVIEW

File: ConsentViewController/Classes/Views/tvOS/NativePrivacyManager/Common/SPNativeScreenViewController.swift [hunk 1]
```diff
@@ -39,14 +39,22 @@ enum SPUIRectEdge {
...
+        var fontSize = spFont?.fontSize != nil ? spFont!.fontSize * magicScalingFactor : UIFont.preferredFont(forTextStyle: .body).pointSize
         let family = spFont?.fontFamily
             .split(separator: ",")
             .map { $0.trimmingCharacters(in: .whitespacesAndNewlines).lowercased() }
             .first { UIFont.familyNames.map { $0.lowercased() }.contains($0) }
             ?? UIFont.systemFont(ofSize: fontSize).familyName
+        if let adjust = adjust {
+            adjust.font = UIFont(name: family, size: fontSize)
+            while (adjust.contentSize.height > adjust.frame.size.height) {
+                print(adjust.contentSize.height.description + ">" + adjust.frame.size.height.description)
```
Comment (andresilveirah) [inline ConsentViewController/Classes/Views/tvOS/NativePrivacyManager/Common/SPNativeScreenViewController.swift:53]: don't forget to remove `print` statements

Comment (Nevazhnovu) [reply ConsentViewController/Classes/Views/tvOS/NativePrivacyManager/Common/SPNativeScreenViewController.swift]: Oops. Fixed

**Comment by Nevazhnovu** [comment]
So all "fitting text" changes were reverted, scrolling applied to description + highlighting added on focus for both CCPA & GDPR first layer views

File: ConsentViewController/Classes/Views/tvOS/NativePrivacyManager/Common/SPFocusableTextView.swift [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    override var canBecomeFocused: Bool { isContentBig }
+    var isContentBig: Bool { self.contentSize.height > self.frame.size.height }
+    public var canFocusCategoryTableView: Bool = true
+
+    open override func didUpdateFocus(in context: UIFocusUpdateContext, with coordinator: UIFocusAnimationCoordinator) {
+        if isFocused {
+            backgroundColor = .lightGray.withAlphaComponent(0.5)
+            flashScrollIndicators()
+            if isContentBig {
+                canFocusCategoryTableView = false
```
Comment (andresilveirah) [inline ConsentViewController/Classes/Views/tvOS/NativePrivacyManager/Common/SPFocusableTextView.swift:20]: @Nevazhnovu having a property called `canFocusCategoryTableView` suggests `SPFocusableTextView` knows about something external to its own responsibilities.

I suggest renaming the field to something like `contentFitsContainer` . The same applies to the method `checkIfCategoryTableViewFocusable`. It should be renamed to something like `updateContentFitsContainer`.

Comment (Nevazhnovu) [reply ConsentViewController/Classes/Views/tvOS/NativePrivacyManager/Common/SPFocusableTextView.swift]: Done

**Pull request merged**